en/08_Changelogs/6.0.0.md

```diff
@@ -1165,6 +1166,14 @@ If you were calling `getSchemaData()` in your `getAttributes()` method in a `For
 
 The related `onAfterRenewToken` extension hook has been renamed to `onAfterRenewSession`, and is triggered at the same logical step in the session renewal process.
 
+### Validation exceptions shows additional info in some contexts {#validation-exception-info}
+
+[`ValidationResult`](api:SilverStripe\Core\Validation\ValidationResult) now includes additional information about validation exceptions including the field name that failed validation, and if applicable the DataObject class and record ID. This additional information will be outputted by [`ValidationException`](api:SilverStripe\Core\Validation\ValidationException) if the validation exception happened in a CLI context, or in an HTTP context if the current controller is an instance of [`DevelopmentAdmin`](api:SilverStripe\Dev\DevelopmentAdmin) or a subclass of it.
```

Comment:
```suggestion
[`ValidationResult`](api:SilverStripe\Core\Validation\ValidationResult) now includes additional information about validation exceptions including the field name that failed validation, and if applicable the model class and record ID. This additional information will be included in the validation message by [`ValidationException`](api:SilverStripe\Core\Validation\ValidationException) if the validation exception happened in a CLI context, or in an HTTP context if the current controller is an instance of [`DevelopmentAdmin`](api:SilverStripe\Dev\DevelopmentAdmin) or a subclass of it.
```
There's no reason to lock this down to `DataObject` specifically.

Reply:
May need to update the conditions noted here depending on code changes in the code PR.

Reply:
Updated